service/ziti-tunnel/service/state.go

```diff
@@ -276,6 +276,10 @@ func (t *RuntimeState) LoadIdentity(id *Id, refreshInterval int) {
 
 		// hack for now - if the identity name is '<unknown>' don't set it... :(
 		if id.CId.Name == "<unknown>" || id.CId.Name == "" {
+			// set name from config, if identity is not authenticated
+			if status != 0 {
```

Comment:
if MFA is enabled, then the events will come with status != 0, and the name in those events are empty. It is difficult to identify the identity if name is not set. So assigned name from config.